src/scenes/AcceleratorRouter/NavBar.tsx

```diff
@@ -68,7 +67,7 @@ export default function NavBar({ backgroundTransparent, setShowNavBar }: NavBarP
         id='overview'
         label={strings.OVERVIEW}
         onClick={() => closeAndNavigateTo(APP_PATHS.ACCELERATOR_OVERVIEW)}
-        selected={!!isOverviewRoute || !!isScoringRoute || !!isVotingRoute || !!isVotingEditRoute}
```

Comment:
@karthikbtf I removed the voting one because the route match seems to be fuzzy, stays highlighted for both `/.../edit` views